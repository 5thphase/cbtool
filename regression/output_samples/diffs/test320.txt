Going to wait until the value on counter "AI (Objects created on the Object Store)" is equal to XX (currently it is equal to XX) waiting 1.0 seconds between samples, for a maximum of 36000.0 seconds. The counter is assumed to be decreasing.
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 0.347179889679 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 1.34717988968 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 2.34717988968 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 3.34717988968 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 4.34717988968 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 5.34717988968 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 6.34717988968 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 7.34717988968 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 8.34717988968 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 9.34717988968 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 10.3471798897 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 11.3471798897 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 12.3471798897 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 13.3471798897 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 14.3471798897 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 15.3471798897 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 16.3471798897 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 17.3471798897 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 18.3471798897 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 19.3471798897 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 20.3471798897 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 21.3471798897 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 22.3471798897 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 23.3471798897 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 24.3471798897 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 25.3471798897 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 1 after 26.3471798897 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 1 after 27.3471798897 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 1 after 28.3471798897 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 1 after 29.3471798897 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 1 after 30.3471798897 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 1 after 31.3471798897 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 1 after 32.3471798897 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 1 after 33.3471798897 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 1 after 34.3471798897 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 0 after 35.3471798897 seconds (the counter is assumed to be decreasing).
Waited 35.3471798897 seconds until "AIs (Objects created on the Object Store)" was equal to 0.
|Redis Version |2.8.6 
|HOSTs |60 
|VMCs ISSUED |16 
|HOSTs ISSUED |0 
|VMs ISSUED |123 
|VMs SLA PROVISIONING VIOLATED |0 
|VMs SLA RUNTIME VIOLATED |0 
|VMs APPLICATION ERRORS |1 
|AIs ISSUED |32 
|AIDRSs ISSUED |1 
|VMCRSs ISSUED |0 
|FIRSs ISSUED |0 
|EXPERIMENT COUNTER |513 
