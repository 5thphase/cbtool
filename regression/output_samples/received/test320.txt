Going to wait until the value on counter "AI (Objects created on the Object Store)" is equal to 0 (currently it is equal to 2) waiting 1.0 seconds between samples, for a maximum of 36000.0 seconds. The counter is assumed to be decreasing.
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 0.462850093842 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 1.46285009384 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 2.46285009384 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 3.46285009384 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 4.46285009384 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 5.46285009384 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 6.46285009384 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 7.46285009384 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 8.46285009384 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 9.46285009384 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 10.4628500938 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 11.4628500938 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 13.4628500938 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 14.4628500938 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 15.4628500938 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 16.4628500938 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 17.4628500938 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 18.4628500938 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 19.4628500938 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 20.4628500938 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 21.4628500938 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 22.4628500938 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 23.4628500938 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 24.4628500938 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 25.4628500938 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 26.4628500938 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 1 after 27.4628500938 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 1 after 28.4628500938 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 1 after 29.4628500938 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 1 after 30.4628500938 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 1 after 31.4628500938 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 1 after 32.4628500938 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 1 after 33.4628500938 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 1 after 34.4628500938 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 1 after 35.4628500938 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 1 after 36.4628500938 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 1 after 37.4628500938 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 1 after 38.4628500938 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 0 after 39.4628500938 seconds (the counter is assumed to be decreasing).
Waited 39.4628500938 seconds until "AIs (Objects created on the Object Store)" was equal to 0.
The following ATTACHED AIs are in this experiment (Cloud TESTCLOUD) :
No objects available.
The following ATTACHED VMs are in this experiment (Cloud TESTCLOUD) :
No objects available.
The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |15.13M                        
|Redis Version                                        |3.0.6                         
|Uptime (in seconds)                                  |8705067                       
|Total Connections Received                           |377371                        
|Total Commands Processed                             |1223224578                    
|Number of Keys                                       |1082                          
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|Collections                                          |15                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |60                            
|VMs                                                  |0                             
|AIs                                                  |0                             
|AIDRSs                                               |1                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ISSUED                                          |16                            
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ISSUED                                         |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |0                             
|VMs ISSUED                                           |123                           
|VMs ARRIVED                                          |119                           
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |119                           
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |4                             
|VMs SLA PROVISIONING VIOLATED                        |0                             
|VMs SLA RUNTIME VIOLATED                             |0                             
|VMs APPLICATION ERRORS                               |1                             
|AI RESERVATIONS                                      |0                             
|AIs ISSUED                                           |32                            
|AIs ARRIVED                                          |30                            
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |30                            
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |2                             
|AIDRS RESERVATIONS                                   |1                             
|AIDRSs ISSUED                                        |1                             
|AIDRSs ARRIVED                                       |1                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ISSUED                                        |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ISSUED                                         |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |513                           

 ---------------- VMC-WIDE COUNTERS ----------------
|48E847B7-4EAD-595E-A603-912566454DDC (simzone_a) VM RESERVATIONS|0                             
|0F125A34-8350-57DE-AA00-635042C08FD4 (simzone_c) VM RESERVATIONS|0                             
|C1CCDA39-30E1-56FF-A6AE-36406AD2BA16 (simzone_d) VM RESERVATIONS|0                             
|67592E97-AC3C-50A4-B3D9-009931854DBD (simzone_b) VM RESERVATIONS|0                             

 ---------------- AIDRS-WIDE COUNTERS ----------------
|3AACD020-5544-50B7-AAE6-CD3A61B3775C (aidrs_1) AI RESERVATIONS|0                             

