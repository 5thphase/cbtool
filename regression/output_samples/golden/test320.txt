Going to wait until the value on counter "AI (Objects created on the Object Store)" is equal to 0 (currently it is equal to 2) waiting 1.0 seconds between samples, for a maximum of 36000.0 seconds. The counter is assumed to be decreasing.
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 0.0687210559845 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 1.06872105598 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 2.06872105598 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 3.06872105598 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 4.06872105598 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 5.06872105598 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 6.06872105598 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 7.06872105598 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 8.06872105598 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 9.06872105598 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 10.068721056 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 11.068721056 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 12.068721056 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 13.068721056 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 14.068721056 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 15.068721056 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 16.068721056 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 17.068721056 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 18.068721056 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 19.068721056 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 20.068721056 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 21.068721056 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 22.068721056 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 23.068721056 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 24.068721056 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 2 after 25.068721056 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 1 after 26.068721056 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 1 after 27.068721056 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 1 after 28.068721056 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 1 after 29.068721056 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 1 after 30.068721056 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 1 after 32.068721056 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 1 after 33.068721056 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 1 after 34.068721056 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 1 after 35.068721056 seconds (the counter is assumed to be decreasing).
 status: Counter "AI (Objects created on the Object Store)" equals 0 after 36.068721056 seconds (the counter is assumed to be decreasing).
Waited 36.068721056 seconds until "AIs (Objects created on the Object Store)" was equal to 0.
The following ATTACHED AIs are in this experiment (Cloud TESTCLOUD) :
No objects available.
The following ATTACHED VMs are in this experiment (Cloud TESTCLOUD) :
No objects available.
The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |23.30M                        
|Redis Version                                        |2.8.6                         
|Uptime (in seconds)                                  |12856679                      
|Total Connections Received                           |548634                        
|Total Commands Processed                             |42589038                      
|Number of Keys                                       |1026                          
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|Collections                                          |28                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |60                            
|VMs                                                  |0                             
|AIs                                                  |0                             
|AIDRSs                                               |1                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ISSUED                                          |16                            
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ISSUED                                         |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |0                             
|VMs ISSUED                                           |123                           
|VMs ARRIVED                                          |119                           
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |119                           
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |4                             
|VMs SLA PROVISIONING VIOLATED                        |0                             
|VMs SLA RUNTIME VIOLATED                             |0                             
|VMs APPLICATION ERRORS                               |1                             
|AI RESERVATIONS                                      |0                             
|AIs ISSUED                                           |32                            
|AIs ARRIVED                                          |30                            
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |30                            
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |2                             
|AIDRS RESERVATIONS                                   |1                             
|AIDRSs ISSUED                                        |1                             
|AIDRSs ARRIVED                                       |1                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ISSUED                                        |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ISSUED                                         |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |513                           

 ---------------- VMC-WIDE COUNTERS ----------------
|A168E857-FDCA-5E02-98BB-32D5CA3934A5 (simzone_d) VM RESERVATIONS|0                             
|2A9C7A1E-D5EE-5CFE-BE56-2B4E298B71F7 (simzone_a) VM RESERVATIONS|0                             
|FB2A884A-73A4-5C91-B913-9AD4AF1FB28A (simzone_c) VM RESERVATIONS|0                             
|073C9CDF-01E7-5AA9-B92E-02C6DA6E75D6 (simzone_b) VM RESERVATIONS|0                             

 ---------------- AIDRS-WIDE COUNTERS ----------------
|6DEFFF8B-03C4-51B2-9A81-F003D4824B76 (aidrs_1) AI RESERVATIONS|0                             

